<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainSession</class>
 <widget class="QWidget" name="MainSession">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>626</width>
    <height>370</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>20</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Sesi√≥n </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Nombre y Apellido</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Velocidad</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Tiempo</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>180</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Pulsaciones</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcdNumber">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>90</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcdNumber_2">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>140</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcdNumber_3">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>140</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>270</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Ver detalles</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>270</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Inicio</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>150</y>
     <width>31</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>150</y>
     <width>21</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>:</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcdNumber_4">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>140</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcdNumber_5">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>180</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
